edb/server/bootstrap.py

```diff
@@ -441,13 +460,13 @@ async def _make_stdlib(testmode: bool, global_ids) -> StdlibBits:
     schema = await _execute_edgeql_ddl(schema, stdglobals)
 
     _, global_schema_version = s_std.make_global_schema_version(schema)
```

Comment:
I think `make_global_schema_version()` here also needs at least the `BackendRuntimeParams` from `ctx` to apply `CreateGlobalSchemaVersion` with the right tenant.

Reply:
No, `make_global_schema_version` returns an abstract DDL command.  `_process_delta` is what compiles it into SQL and that gets the tenant id properly.